recon_linux.go

```
@@ -43,17 +44,16 @@ func (irc *IrcCon) StartUnixListener() {
 }
 
 // Attempt to hijack session previously running bot
-func (irc *IrcCon) HijackSession() bool {
+func (irc *Bot) hijackSession() bool {
 	unaddr, err := net.ResolveUnixAddr("unix", irc.unixastr)
 	if err != nil {
-		irc.Log(LWarning, "could not resolve unix socket")
+		irc.Error("could not resolve unix socket")
```

yeah, i would. just for sanity. Imagine it changes underneath us and we are ignoring the error.